docs/deploying_apps/deploying_static_sites.md

```diff
@@ -1,82 +1,50 @@
-Create `index.html`:
+This section explains how to create and deploy a static HTML page. It's
+worth testing that you can carry out this process before you try to deploy a dynamic app.
 
-```
-$ touch index.html
-```
+These steps assume you have already carried out the setup process explained in the [Quick Setup Guide](/getting_started/quick_setup_guide) section.
 
-Add some markup:
+1. Create `index.html`:
 
-```html
-<html>
-  <head>
-    <title>Static Site</title>
-  </head>
-  <body>
-    <p>Welcome to the static site!</p>
-  </body>
-</html>
-```
+    ``
+    touch index.html
+    ``
 
-Create a `manifest.yml` that uses the [`staticfile_buildpack`](https://github.com/cloudfoundry/staticfile-buildpack):
+2. Add some markup to the `index.html` file:
 
-```yml
----
-applications:
-- name: my-static-site
-  memory: 64M
-  buildpack: staticfile_buildpack
-```
+    
+        <html>
+          <head>
+            <title>Static Site</title>
+          </head>
+          <body>
+            <p>Welcome to the static site!</p>
+          </body>
+        </html>
+    
 
-If the static content is included in a different folder, you can add a `path` declaration. E.g., `path: dist` or `path: assets`.
+3. Create a `manifest.yml` which uses the [`staticfile_buildpack`](https://github.com/cloudfoundry/staticfile-buildpack):
 
-Deploy:
+        ---
+        applications:
+        + name: my-static-site
+        memory: 64M
+        buildpack: staticfile_buildpack
+    
+    Replace ``my-static-site`` with the name you want for your app.
```

Comment:
Might be worth nothing that if the name is not unique you may get an error message, for example:

FAILED
Server error, status code: 400, error code: 210003, message: The host is taken: healthcheck